components/button/Button.tsx

```diff
@@ -4,7 +4,7 @@ interface ButtonProps {
   /**
    * The code to call on button click
    */
-  action?: () => null;
+  action?: () => void;
```

Comment:
Why you changed that?

Reply:
Because i had this error before: Type '() => Promise<void>' is not assignable to type '() => null'.
  Type 'Promise<void>' is not assignable to type 'null'.